pkg/querier/tripperware/instantquery/instant_query_test.go

```diff
@@ -323,7 +330,7 @@ func TestResponse(t *testing.T) {
 			promBody: &tripperware.PrometheusResponse{
 				Status: "success",
 				Data: tripperware.PrometheusData{
-					ResultType: model.ValString.String(),
+					ResultType: model.ValScalar.String(),
```

Comment:
I wonder if we missed something in the test case itself so we didn't really catch this issue on result type. Should we enhance our tests?

Reply:
It could be checked if we add a validation logic to `DecodeResponse` or `EncodeResponse`. I think it is just a typo.